src/module-mui.d.ts

```diff
@@ -25,9 +24,9 @@ declare module '@mui/material/styles/createTheme' {
         };
         agGridBackground: CSSObject;
     };
-    export interface Theme extends MuiTheme, ThemeExtension {}
+    export interface Theme extends /*MuiTheme,*/ ThemeExtension {}
     // allow configuration using `createTheme`
-    export interface ThemeOptions extends MuiThemeOptions, Partial<ThemeExtension> {}
+    export interface ThemeOptions extends /*MuiThemeOptions,*/ Partial<ThemeExtension> {}
```

Comment:
why ?

Reply:
<pre>&gt; tsc &amp;&amp; vite build

<font color="#34E2E2">src/module-mui.d.ts</font>:<font color="#FCE94F">27</font>:<font color="#FCE94F">22</font> - <font color="#EF2929">error</font><font color="#555753"> TS2310: </font>Type &apos;Theme&apos; recursively references itself as a base type.

<span style="background-color:#FFFFFF"><font color="#300A24">27</font></span>     export interface Theme extends MuiTheme, ThemeExtension {}
<span style="background-color:#FFFFFF"><font color="#300A24">  </font></span> <font color="#EF2929">                     ~~~~~</font>

<font color="#34E2E2">src/module-mui.d.ts</font>:<font color="#FCE94F">29</font>:<font color="#FCE94F">22</font> - <font color="#EF2929">error</font><font color="#555753"> TS2310: </font>Type &apos;ThemeOptions&apos; recursively references itself as a base type.

<span style="background-color:#FFFFFF"><font color="#300A24">29</font></span>     export interface ThemeOptions extends MuiThemeOptions, Partial&lt;ThemeExtension&gt; {}
<span style="background-color:#FFFFFF"><font color="#300A24">  </font></span> <font color="#EF2929">                     ~~~~~~~~~~~~</font>


Found 2 errors in the same file, starting at: src/module-mui.d.ts<font color="#555753">:27</font>
</pre>